apps/src/code-studio/components/GridEditor.jsx

```diff
@@ -150,7 +151,7 @@ var GridEditor = React.createClass({
       }, this);
     }, this);
 
-    var serializedData = cells.map(function (row) {
+    const serializedData = cells.map(function (row) {
       return row.map(function (cell) {
         return cell.serialize();
       });
```

Comment:
here and lines 145-148 are more places we could use arrow funcs. Not sure how completely you're trying to ES6ify (will prob stop commenting on this if there are other places like it).